src/com/content-games/games.js

```diff
@@ -9,10 +9,10 @@ import ContentItemBox					from 'com/content-item/item-box';
 import ContentCommonBody				from 'com/content-common/common-body';
 import ContentCommonBodyTitle			from 'com/content-common/common-body-title';
 
-import GridSelector						from './grid-selector';
-
 import ContentMore						from 'com/content-more/more';
 
+import SelectableGrid 							from 'com/layout/grid/selectable-grid';
```

Comment:
Prefer `CardGrid` (Cards being the new term for things currently known as `boxes`).

Reply:
So would that be `SelectableCardGrid` then? Given that there's a grid component separately and this   one is just with the selector.

Reply:
What makes it selectable?